### PR TITLE
Homework One

### PR DESCRIPTION
**Pull request by GabrielKahn**
There were several steps to successfully completing this assignment, some harder than others. As I already had R Studio installed on the computer, the first obstacle I encountered was installing GIT and linking it to my R Studio. After a period of trouble shooting, I overcame this problem by giving R Studio the correct path to GIT.

Pulling the assignment from GitHub was relatively simple, as was completing the about me section of the homework. Luckily I had the assistance of an R-Markdown cheat sheet from another class which greatly assisted me in creating the proper layout of the homework. Probably the most fun part of the assignment was messing around with the different tools to alter the text and create lists.

Committing the changes from R-Studio to GitHub went relatively smoothly, but the process is still a bit confusing for me. Deb was a great help for this part of the assignment. She helped guide me through some extra steps I had to undertake, including putting my credentials into GitBash. She also helped show me how to push my changes into GitHub so that my final work shows up.

As a general reflection, I feel relatively at ease using R Markdown, but I still need to get more comfortable with the Pull/Commit/Push process working between R Studio and GitHub. After I finish with this write up, I am going to do more research into how everything works, and also view how my homework shows up on the class page in GitHub. There is much I need to learn still about this process and I need to get over my fear of losing my work, but now that I have completed my first assignment in GitHub I expect that I will be more comfortable with the interface going forward. While it is still a mystery to me how my R Studio communicates with GitHub, at least I got everything to work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+
+Additionally, here is a picture of my favorite animal
+
+![Tiger](http://goodnature.nathab.com/wp-content/uploads/2014/07/7465214366_5d0cb50707_k.jpg)
+
+That's enough facts for now, but there are many more.
+
+Here is an excellent [site](https://cfss.uchicago.edu) if you want to learn more about how I created this document.
+
+
```
Comment (whitingam) [inline README.md:19]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

I think it's great that you used all 5 elements of markdown, I didn't get the links to work as I should have apparently!  Your tiger picture is also really awesome.  Really great job, I can't find anything to criticize.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
-# hw01
-Repository for homework 01
```
Comment (jfan3) [inline README.md:2]: Nice!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+# Homework One
+*by Gabriel Kahn*
```
Comment (jfan3) [inline README.md:2]: Nice!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,19 @@
...
+
+Hello my name is Gabriel Kahn and I am **very** excited to be taking this course with all of you. There are lots of fun facts of abut me. Let's make a list:
```
Comment (jfan3) [inline README.md:4]: Can create more sections around here.

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |x           |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |            |x              |            |
| **Achievement, creativity** |           |x              |            |
| **Ease of access**          |x           |              |            |

Remarks:

* Nice work!
* Could use a more sectional structure.

**Pull request closed without merging**